Tests/ApolloTests/VersionNumberTests.swift

```diff
@@ -4,6 +4,6 @@ import XCTest
 class VersionNumberTests: XCTestCase {
   func testVersionNumberExists() {
     // It would be the first 2 digits of version number like `0.19`.
-    XCTAssertGreaterThanOrEqual(ApolloVersionNumber, 0)
+//    XCTAssertGreaterThanOrEqual(ApolloVersionNumber, 0)
```

Comment:
This should not be committed - I know it doesn't run right if you're not using the schemes, but I'd still prefer to run it with the schemes.

Reply:
Yep, this was my bad, I meant to uncomment and should be back to normal now